### ISSUE TITLE
[Epic] Identity list of known bugs related to 8.7 reachitecture 

### ISSUE THREAD

**Issue by mschoe** [camunda/camunda#26059]
# Description 
This issue is a reference to discovered bugs that affects the Identity 8.7 re-architecture, which need to be fixed until the release.

**Comment by npepinpe** [comment]
@ThorbenLindhauer @mschoe - are we still tracking this? Or is this superseded by other issues?

**Comment by ThorbenLindhauer** [comment]
I have never seen this ticket before, I think we can close it. I believe there are more that were created in Q3/Q4 2024 and are not relevant anymore since Meggle created the milestone breakdown with new epics. For example [this one](https://github.com/camunda/camunda/issues/19670) and others on the board here: https://github.com/orgs/camunda/projects/120/views/6

**Comment by npepinpe** [comment]
OK, who would best know what can be closed? I'd like to get as clean as start when doing planning and make sure we aren't forgetting anything. I understand this is a bit time consuming, so no need to do this immediately while we're all focused on code freeze.

I can do a first pass on what is obvious, and then also ping you when I'm not sure.

**Comment by ThorbenLindhauer** [comment]
Yeah, I think I can do that best. Feel free to start and send me anything that's not clear to you. If in doubt I'd prioritize code-freeze-related things until then, but at latest past that point we can clean it all up.
